embassy-stm32/build.rs

```diff
@@ -483,7 +483,9 @@ fn main() {
         (("i2c", "TX"), quote!(crate::i2c::TxDma)),
         (("dcmi", "DCMI"), quote!(crate::dcmi::FrameDma)),
         (("dcmi", "PSSI"), quote!(crate::dcmi::FrameDma)),
-        (("sdmmc", "SDIO"), quote!(crate::sdmmc::SdmmcDma)),
+        // Same channel is used for RX/TX
+        (("sdmmc", "RX"), quote!(crate::sdmmc::SdmmcDma)),
+        (("sdmmc", "RX"), quote!(crate::sdmmc::SdmmcDma)),
```

Comment:
RX->TX?

Reply:
SDMMCv1 uses the same DMA channel for both directions. However, `stm32-data` generates 3 signals `RX`, `TX` and `SDIO` (or `SDMMCx`) for the same DMA channel so I just picked `RX`. Maybe this could also be fixed in `stm32-data`, but idk how you call it?

Reply:
ohhh, sorry I saw the problem xD